src/console_cmds.cpp

```diff
@@ -1308,7 +1309,21 @@ DEF_CONSOLE_CMD(ConGetDate)
 
 	YearMonthDay ymd;
 	ConvertDateToYMD(_date, &ymd);
-	IConsolePrintF(CC_DEFAULT, "Date: %d-%d-%d", ymd.day, ymd.month + 1, ymd.year);
+	IConsolePrintF(CC_DEFAULT, "Date: %d-%02d-%02d", ymd.day, ymd.month + 1, ymd.year);
+	return true;
+}
+
+DEF_CONSOLE_CMD(ConGetSysDate)
+{
+	if (argc == 0) {
+		IConsoleHelp("Returns the current date (day-month-year) of your system. Usage: 'getsysdate'");
+		return true;
+	}
+
+	time_t t;
+	time(&t);
+	auto timeinfo = localtime(&t);
+	IConsolePrintF(CC_DEFAULT, "System Date: %d-%02d-%02d %d:%d:%d", timeinfo->tm_year + 1900, timeinfo->tm_mon + 1, timeinfo->tm_mday, timeinfo->tm_hour, timeinfo->tm_min, timeinfo->tm_sec);
```

Comment:
Minute and second should also use `%02d` to look right, as otherwise you get `8:5:7` for 5 minutes and 7 seconds past 8 in the morning.

Reply:
I'd use `%04d` for year, even if it's basically already 4 digits, just for extra consistency.

Reply:
Thanks, all done.